man/check-os-release.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -eu
```

Comment:
commit msg says "bash", but this declares any bourne shell. which one is right?

Reply:
`sh`. I'll adjust the commit message.